src/scss/_navbar.scss

```diff
@@ -1,11 +1,20 @@
 .navbar {
   position: fixed;
+  height: var(--header-height);
+  width: 100%;
+  padding: 0;
+  overflow: hidden;
```

Comment:
I suppose it's caused by this hidden overflow. @hoetmaaiers do you remember why this was added?

Reply:
I can't directly remember why this was added. But the mobile dropdown is blamed by it? Is an other issue already registered for this?

Reply:
Yes, see #304 for the issue, and #313 for a potential fix.

Also other dropdowns in the navbar (in non-mobile mode) are not working because of this.